client/src/js/components/bhModalNotify/bhModalNotify.html

```diff
@@ -0,0 +1,21 @@
+<style>
+  .bh-modal-notify {
```

Comment:
I can tell you've been writing VueJS.  Can you just put this in our bhima-bootstrap stylesheet?